task_2.py

```diff
@@ -0,0 +1,14 @@
+# Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+
+def double_list(array: list[int]) -> list[int]:
+    res = set()
+    for el in array:
+        counter = array.count(el)
+        if counter > 1:
+            res.add(el)
```

Comment:
У вас выходит квадратичная сложность? Но можно немного уменьшить.
```suggestion
    res = []
    for el in set(array):
        counter = array.count(el)
        if counter > 1:
            res.append(el)
```